### PR TITLE
GDB-5192 fix cypress tests run

### PR DESCRIPTION
**Pull request by svilenvelikov**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
         "test": "karma start",
         "test:coverage": "rimraf coverage/ && karma start --single-run --browsers ChromeHeadless",
         "test:coveralls": "cat coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
-        "test:acceptance": "scripts/run-cypress-tests.sh $npm_package_versions_graphdb",
+        "test:acceptance": "export $(cat .env | xargs) && scripts/run-cypress-tests.sh",
```
Comment (mihailradkov) [inline package.json:12]: Just for the record, we can use https://www.npmjs.com/package/dotenv for dealing with .env files.

File: scripts/run-cypress-tests.sh [hunk 2]
```diff
@@ -5,7 +5,9 @@ set -eu
 
 echo "Working directory: $(pwd)"
 
-GRAPHDB_VERSION=$1
+GRAPHDB_VERSION=${GDB_VERSION}
```
Comment (mihailradkov) [inline scripts/run-cypress-tests.sh:8]: No point in having the extra variable.

**Review by mihailradkov — DISMISSED** [review]

File: package.json (hunk 1 above)
Comment (svilenvelikov) [reply package.json:12]: Yes, we can. For now it's not really needed.

File: scripts/run-cypress-tests.sh (hunk 2 above)
Comment (svilenvelikov) [reply scripts/run-cypress-tests.sh]: removed

**Review by desislava-hristova-ontotext — APPROVED** [review]

**Review by desislava-hristova-ontotext — CHANGES_REQUESTED** [review]
Change the version to 1.5.0-TR10 as in the master

**Review by desislava-hristova-ontotext — APPROVED** [review]

**Pull request merged**